src/nhsd/components/molecules/filter-menu-section/filter-menu-section.js

```diff
@@ -0,0 +1,27 @@
+export default class NHSDFilterMenu {
+  constructor(componentEl) {
+    if (!componentEl) return null;
+    this.hostEl = componentEl;
+    this.init();
+  }
+
+  /* eslint-disable */
```

Comment:
Is there a reason why eslint has been disabled here? I think you should be able to write this block without disabling eslint.